6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<p>Print out the values here. People shouldn't have to scroll up to see what the `p_waic` values were for each model.</p><p><br></p><blockquote>However, when it comes to performance,</blockquote><p>What do you mean by performance? You mention it here, but someone may not know to what you are referring. </p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Made some changes. I replaced the mention of 'perform' everywhere with 'fit', 'goodness' and 'compatible'. These words were used to explain the concept of model comparison at the beginning.

It now looks like:
# Interpretation
Since the hierarchical model seems to be more complex and flexible, one might expect it to fit better on our data. It does have higher complexity which can be seen by comparing the p_waic values for both the models.

Block for printing the p_pooled, p_hierarchical values.

So, we get:
`p_hierarchical` > `p_pooled`
However, when it comes to deciding which model better fits the data ('goodness' of the model, as discussed earlier), we can see that the computed `waic` values for the models are approximately equal.

Block for printing the p_pooled, p_hierarchical values.

This means that both models are equally compatible with our data. Hence, we may be tempted to use the more complex model but this example shows that the simpler pooled model compares equally well.